daps-xslt/migrate/db5to4/db4tonovdoc.xsl

```diff
@@ -950,4 +950,68 @@
     </tbody>
   </xsl:template>
 
+  <!-- Novdoc does not allow affiliation, but we can do tricks to pull that
+  info into one of the name fields (here: surname). To make sure that we
+  always get same layout, we just try stuff everything in the same tag. -->
+  <xsl:template match="*[self::author or self::collab or
+    self::editor or self::othercredit][affiliation]">
```

Comment:
That XPath looks a bit weird to me. Shouldn't that XPath a bit more specific? Something like this:

```
(author | collab | editor | othercredit)[affiliation]
```

However, I haven't tested your stylesheet. 😉

Reply:
*facepalm* yeah, you're right.